Reference/google/protobuf/any_test.pb.swift

```diff
@@ -107,6 +107,7 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Proto3Message, SwiftProtobuf._Mes
   init() {}
 
   mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
+    _ = _uniqueStorage()
     try withExtendedLifetime(_storage) { (_storage: _StorageClass) in
```

Comment:
After thinking about it over the weekend, I'm not entirely clear on why the `withExtendedLifetime` helps here; I've not come up with any scenario where `_storage` could get released while this method is running.